packages/calcite-components/src/components/input-date-picker/input-date-picker.scss

```diff
@@ -91,7 +104,7 @@
 .divider {
   @apply inline-block w-px;
```

Comment:
divider has a 1px width: ` @apply inline-block w-px;`

Should that be a var?

<img width="337" alt="image" src="https://github.com/user-attachments/assets/125cf435-6511-4f30-8bbe-5ed2087ede93" />

Reply:
Same with divider-container